### PR TITLE
address feedback from original PR

### PR DESCRIPTION
**Pull request by dorothy-zbornak**
## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: contracts/zero-ex/contracts/src/features/ERC721OrdersFeature.sol [hunk 1]
```diff
@@ -589,66 +589,65 @@ contract ERC721OrdersFeature is
             .preSigned[orderHash][msg.sender] = true;
     }
 
-    // Core settlment logic for selling an ERC721 asset. 
-    // Used by `sellERC721` and `onERC721Received`. 
+    // Core settlment logic for selling an ERC721 asset.
```
Comment (kimpers) [inline contracts/zero-ex/contracts/src/features/ERC721OrdersFeature.sol:592]:
```suggestion
    // Core settlement logic for selling an ERC721 asset.
```

File: contracts/zero-ex/contracts/src/features/ERC721OrdersFeature.sol [hunk 2]
```diff
@@ -278,41 +273,37 @@ contract ERC721OrdersFeature is
         _validateBuyOrder(
             buyOrder,
             buyOrderSignature,
-            msg.sender, // TODO: maybe allow sellOrder.taker == buyOrder.maker, vice versa
+            sellOrder.maker,
```
Comment (mzhu25) [inline contracts/zero-ex/contracts/src/features/ERC721OrdersFeature.sol:276]: If we're going to do this we should also pass in `buyOrder.maker` to `_validateSellOrder`

File: contracts/zero-ex/contracts/src/features/ERC721OrdersFeature.sol [hunk 3]
```diff
@@ -589,66 +589,65 @@ contract ERC721OrdersFeature is
...
+    // Used by `sellERC721` and `onERC721Received`.
     function _sellERC721(
-        LibERC721Order.ERC721Order memory order,
+        LibERC721Order.ERC721Order memory buyOrder,
         LibSignature.Signature memory signature,
         uint256 erc721TokenId,
         bool unwrapNativeToken,
         address taker,
-        bool isCallback,
+        address nftOwner,
```
Comment (mzhu25) [inline contracts/zero-ex/contracts/src/features/ERC721OrdersFeature.sol:600]: `currentOwner` perhaps?

**Review by mzhu25 — COMMENTED** [review]
Thanks for this
~~I'm also going to rename `_sellERC721` -> `_executeERC721BuyOrder` and `_buyERC721` -> `_executeERC721SellOrder`~~ actually I change my mind. idk

**Pull request merged**